tsconfig.json

```diff
@@ -8,5 +8,8 @@
     "incremental": true,
     "moduleResolution": "node",
     "strict": true
-  }
+  },
+  "include": [
+    "src/**/*"
+  ]
```

Comment:
When having TypeScript files under the `test` directory, we can no longer rely on the `tsc` heuristic to determine the source root directory. Otherwise `npm run build:types` will produce subfolders `src` and `test` in the output directory instead of putting the content of `src` in the output directory.